core/common/scale/advanced.cpp

```diff
@@ -0,0 +1,9 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "common/scale/advanced.hpp"
+
+namespace kagome::common::scale::advanced {
+}  // namespace kagome::common::scale::advanced
```

Comment:
why do you need cpp file here? it is empty

Reply:
Possibly not really, I will remove it later if it comes to be sure useless.